src/Tickets/Emails/Email/Ticket.php

```diff
@@ -272,37 +171,53 @@ public function get_default_subject_multiple() {
 	}
 
 	/**
-	 * Get email content.
-	 *
-	 * @since 5.5.9
-	 *
-	 * @param array $args The arguments.
-	 *
-	 * @return string The email content.
+	 * Get subject for plural tickets.
+	 * 
+	 * @since TBD
+	 * 
+	 * @return string
 	 */
-	public function get_content( $args = [] ): string {
-		// @todo: Parse args, etc.
-		$context = ! empty( $args['context'] ) ? $args['context'] : [];
+	public function get_subject_plural(): string {
+		$option_key = $this->get_option_key( 'subject-plural' );
+		$subject = tribe_get_option( $option_key, $this->get_default_subject_plural() );
 
-		// @todo: We need to grab the proper information that's going to be sent as context.
+		// @todo: Probably we want more data parsed, or maybe move the filters somewhere else as we're always gonna
 
-		$email_template = tribe( Email_Template::class );
-		$email_template->set_preview( true );
+		/**
+		 * Allow filtering the email subject globally.
+		 *
+		 * @since TBD
+		 *
+		 * @param string $subject  The email heasubjectding.
+		 * @param string $id       The email id.
+		 * @param string $template Template name.
+		 */
+		$subject = apply_filters( 'tec_tickets_emails_subject_plural', $subject, $this->id, $this->template );
```

Comment:
I think we're only going to have plural subjects for Tickets and RSVPs, not sure if we need to have a global filter. We might if we repeat this in the RSVP, though.

Reply:
Right. I was thinking about RSVPs, actually.